test/unit/resources/aws_elasti_cache_cluster_node_test.rb

```diff
@@ -0,0 +1,84 @@
+require 'helper'
+require 'aws_elasti_cache_cluster_node'
+require "aws-sdk-core"
+require_relative 'mock/aws_elasti_cache_cluster_mock'
+
+# class AwsElastiCacheClusterNodeConstructorTest < Minitest::Test
```

Comment:
Are these lines supposed to be commented?

Reply:
No 😷